styles/layouts/scss/rooms.scss

```diff
@@ -30,6 +30,8 @@
     border-radius: 10px;
     margin: 10px;
     padding: 20px;
+    flex-wrap: nowrap;
+    justify-content: space-between;
```

Comment:
Two lines of code were added to display three elements in a row.

Reply:
Thanks 😃


It will be very interesting to see how the class will look on the page you working on with the full list. I've already taken a quick look and noticed that there will be some issues with the CSS 😅